### DISCUSSION TITLE
Error: No matching key pair found in [object Object]

### DISCUSSION THREAD

**Discussion by rulik55** [near/wallet-selector#1078] [General]
Error: No matching key pair found in [Object].

I tried logging in from different browsers (I used Google Chrome, Brave, Opera and others), from different devices (computers, phones), from different wallets (exported the phrase).
I cleared the cache, deleted the account and installed again. I tried installing new keys. Nothing helps. Tried to do this many times!
I can’t connect to the application, send nearby, change keys and other actions. I keep getting the error: No matching key pair found in [object object].

I already wrote to support, I wrote that the instructions from the site Nearhelp.zendesk.com did not help. In response, I received the same instructions. What should I do if these instructions do not help?

There are no more replies to the letter, as well as a solution to the problem.

This issue occurs in two wallets (Error: No matching key pair found in [object object]):

The problem cannot be solved using the instructions on closehelp.zendesk.com.
The instructions I received are the same as the instructions on the website.
I duplicate the instructions received, they do not help in solving the problem!

We have identified several possible reasons for the error you are experiencing. To begin troubleshooting, consider the following steps:
 
Check your browser: Transactions generally work best in Google Chrome or Brave browsers. Make sure you are using one of these browsers and it is important to stay in the same browser without clearing the cache when performing transactions. This error may occur due to a locally stored key pair mismatch.
Try the following steps:
Remove your account from your wallet: Simply click the red button located in the Account section.
Clear your browser cache.
Re-import your account. For detailed instructions on importing your account, check out our helpful guide.
Check the accuracy of the passphrase entered and note that the exact order is critical.
Account removed from wallet
If you have successfully deleted the account from your wallet (see screenshot below), you should no longer have any problems.
If you haven't deleted your account from the wallet (as shown in the screenshot below), you can try steps 1 and 2 above to fix the situation.
![Снимок экрана 2024-02-13 214713](https://github.com/near/DX/assets/130463557/f31be936-d1ad-45ef-ba97-685d3c28ec6a)

**Comment by thisisjoshford** [comment]
Hey @rulik55 ! Sorry you're experiencing issues with your wallet. I transferred this discussion to the wallet-selector repo as its the closest in the NEAR github org to your issue. It looks like you are using MyNEARWallet and would suggest reaching out to that team directly: [support@mynearwallet.com](mailto:support@mynearwallet.com) I was going to suggest filing an issue on their repo directly, but I'm not seeing that option there (https://github.com/mynearwallet/my-near-wallet).

**Reply by rulik55** [reply]
Importing the wallet seed phrase does not work in any application.

**Reply by rulik55** [reply]
![Screenshot_2024-02-14-08-07-05-845_com herewallet](https://github.com/near/wallet-selector/assets/130463557/8aeda409-a341-40c7-878a-f03b730476aa)
![Screenshot_2024-02-14-08-07-11-692_com herewallet](https://github.com/near/wallet-selector/assets/130463557/6961cb12-fc20-49dd-bbb9-8499d7dda5ea)
Here is a screenshot from Here wallet.
